app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt

```diff
@@ -39,7 +39,7 @@ class BindableAdapter<T : Any> internal constructor(
   // TODO(#170): Introduce support for stable IDs.
 
   /** Sets the data of this adapter. This is expected to be called by Android via data-binding. */
-  private fun setData(newDataList: List<T>) {
+  fun setData(newDataList: List<T>) {
```

Comment:
Please note that to attach the slides to the adapter, I had to make this public.

Reply:
This seems reasonable--if we know the type & can set it safely we should (vs. using setDataUnchecked).